### ISSUE TITLE
a found in  binding.NewBindings

### ISSUE THREAD

**Issue by abin30** [wailsapp/wails#1925]
### Description

in code file  github.com/wailsapp/wails/v2/internal/binding/binding.go ,I found these code 
`
func NewBindings(logger *logger.Logger, structPointersToBind []interface{}, exemptions []interface{}, obfuscate bool) *Bindings {
          ...
	for _, exemption := range exemptions {
		if exemptions == nil {                   
			continue
		}
	....
	}

`
		if exemptions == nil {                   =>	if exemption == nil { 
`

### To Reproduce

1. open "github.com/wailsapp/wails/v2/internal/binding/binding.go"
2. line 38    "		if exemptions == nil {"

### Expected behaviour

"if exemptions == nil  " will be     "		if exemption == nil {"

### System Details

```shell
Wails CLI v2.0.0-rc.1.1

Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:             Windows 10 Enterprise
Version:        2009 (Build: 19042)
ID:             20H2
Go Version:     go1.18
Platform:       windows
Architecture:   amd64

Wails
------

Diagnosis
---------
Your system is ready for Wails development!
```

**Comment by leaanthony** [comment]
Well spotted! 👏 Feel free to press "." In GitHub and send a fix. If I don't hear anything then I'll just fix myself. Thank you 🙏

**Comment by stffabi** [comment]
Thanks for bringing this up. The fix has been added to master with 6a3b89d212fb23e47b24e10ea21aa95988d04b90
